app/views/layouts/_nav.html.erb

```diff
@@ -1,8 +1,7 @@
 <div class="prx-header-bar | navbar navbar-dark align-items-stretch p-0 bg-primary-dark">
   <div class="container-fluid align-items-stretch pe-0">
     <header class="flex-grow-1">
-      <h1 class="visually-hidden">PRX Dovetail</h1>
```

Comment:
Just an FYI - I'm not a fan of using a Header for the Site logo. Bootstrap doesn't suggest one either. The link is also self described.